assets/js/modules/analytics-4/datastore/__fixtures__/index.js

```diff
@@ -23,6 +23,7 @@ export { default as conversionEvents } from './conversion-events.json';
 export { default as createProperty } from './create-property.json';
```

Comment:
Please remember to review the notes in **Relevant technical choices** in the PR description.